mobile_verifier/migrations/15_speedtests_one_to_one.sql

```diff
@@ -0,0 +1,18 @@
+
+CREATE TABLE speedtests_migration (
+       pubkey text NOT NULL,
+       upload_speed bigint,
+       download_speed bigint,
+       latency integer,
+       timestamp timestamptz NOT NULL
+);
+CREATE INDEX idx_speedtests_pubkey on speedtests_migration (pubkey);
+
+INSERT INTO speedtests_migration (pubkey, upload_speed, download_speed, latency, timestamp)
+SELECT id, (st).upload_speed, (st).download_speed, (st).latency, (st).timestamp
+FROM (select id, unnest(speedtests) as st from speedtests) as tmp;
+
+ALTER TABLE speedtests RENAME TO speedtests_old;
```

Comment:
I should change this to drop the old table but for safety sake renaming for now.  Will change to drop before merge

Reply:
i am grand with that